script/cibuild-gh-ost-replica-tests

```diff
@@ -36,8 +36,16 @@ test_mysql_version() {
 
   mkdir -p sandbox/binary
   rm -rf sandbox/binary/*
-  gh-ost-ci-env/bin/linux/dbdeployer unpack gh-ost-ci-env/mysql-tarballs/"$mysql_version".tar.xz --sandbox-binary ${PWD}/sandbox/binary
-
+  local mysql_server=${mysql_version%-*}
+  if echo "$mysql_server" | egrep -i "percona" ; then
+    tarball_name=Percona-Server-${mysql_version#*-}-12-Linux.x86_64.glibc2.12-minimal.tar.gz
+    rm -f gh-ost-ci-env/mysql-tarballs/${tarball_name}
+    ln -s "$mysql_version".tar.xz gh-ost-ci-env/mysql-tarballs/${tarball_name}
+    gh-ost-ci-env/bin/linux/dbdeployer unpack gh-ost-ci-env/mysql-tarballs/${tarball_name} --sandbox-binary ${PWD}/sandbox/binary
+    rm -f gh-ost-ci-env/mysql-tarballs/${tarball_name}
+  else
+    gh-ost-ci-env/bin/linux/dbdeployer unpack gh-ost-ci-env/mysql-tarballs/"$mysql_version".tar.xz --sandbox-binary ${PWD}/sandbox/binary
+  fi
```

Comment:
I'm not sure if these changes are necessary - in CI `mysql_version` will be set to `PerconaServer-8.0.21` so should work with the original code?

Reply:
> I'm not sure if these changes are necessary - in CI `mysql_version` will be set to `PerconaServer-8.0.21` so should work with the original code?

Since the tarball name of PerconaServer-8.0.21 in dbdeployer is `Percona-Server-8.0.21-12-Linux.x86_64.glibc2.12-minimal.tar.gz`, dbdeployer unpack tarball "PerconaServer-8.0.21.tar.xz" failed with the origin code.